eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -985,6 +985,26 @@ stages:
       - template: ../steps/vmr-validate-asset-baseline.yml
         parameters:
           continueOnError: true
+    - job: ValidateInstallers_Linux_x64
+      displayName: Validate Installers - Linux x64
+      pool: ${{ parameters.pool_Linux }}
+      timeoutInMinutes: 240
```

Comment:
This timeout seems excessive.  Looks like in your test run it took 20 minutes.

Reply:
Yeah, I'll adjust - thanks.

Reply:
Fixed with https://github.com/dotnet/sdk/pull/47313/commits/d9895e41c6e5fd6481b02d77f7dbe6a9bd7efec0